dashboard/app/models/user.rb

```diff
@@ -124,8 +124,6 @@ class User < ActiveRecord::Base
 
   acts_as_paranoid # use deleted_at column instead of deleting rows
 
-  scope :ignore_deleted_at_index, -> {from 'users IGNORE INDEX(index_users_on_deleted_at)'}
```

Comment:
Just curious, why this change?

Reply:
Not sure what happened here - seems like something happened with Git along the way.